openedx/core/djangoapps/credentials/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
```

Comment:
The purpose of this app will be to serve the requests from edx-platform and after some processing/validation on them pass it to the service. This is added here as we are planing to move the service in a separate IDA so this will act as way of communication between edx-platform and IDA.